pkg/api/v1/types.go

```diff
@@ -171,6 +173,31 @@ type Affinity struct {
 	PodAntiAffinity *k8sv1.PodAntiAffinity `json:"podAntiAffinity,omitempty"`
 }
 
+// The vmi this Toleration is attached to tolerates any taint that matches
+// the triple <key,value,effect> using the matching operator <operator>.
+// +k8s:openapi-gen=true
+type Toleration struct {
```

Comment:
You can directly use the struct in `/home/rmohr/go/src/kubevirt.io/kubevirt/vendor/k8s.io/api/core/v1/types.go`. We do the same for other structs too which exactly match the pod struct.

Reply:
done, but the only problem here is the swagger doc description will be a little wrong since it's `vmi` not `pod`, also if you think this change is okay, I guess we should also change `type Affinity struct {`